qcodes/dataset/plotting.py

```diff
@@ -310,11 +311,14 @@ def plot_by_id(run_id: int,
                complex_plot_phase: str = 'radians',
                **kwargs) -> AxesTupleList:
     """
-    Construct all plots for a given `run_id`. All other arguments are forwarded
+    Construct all plots for a given `run_id`. Here `run_id` is an
+    alias for `captured_run_id` for historical reasons. See the docs
```

Comment:
They do not currently use the captured counter. The long term plan is to get rid of them completely

Reply:
exactly, but then perhaps it's nice to add a note there that they don't use "captured_run_id", but "run_id". just because "long term plan"s have the tendency to be quite long term ((